aloha/real_env.py

```diff
@@ -172,29 +186,35 @@ def get_base_vel(self):
         angular_vel = self.base.base.get_angular_velocity().z
         return np.array([linear_vel, angular_vel])
 
-    def set_gripper_pose(
-        self,
-        left_gripper_desired_pos_normalized,
-        right_gripper_desired_pos_normalized
-    ):
-        left_gripper_desired_joint = FOLLOWER_GRIPPER_JOINT_UNNORMALIZE_FN(
-            left_gripper_desired_pos_normalized
-        )
-        self.gripper_command.cmd = left_gripper_desired_joint
-        self.follower_bot_left.gripper.core.pub_single.publish(self.gripper_command)
+    def set_gripper_pose(self, robot_name: str, gripper_desired_pos_normalized: float) -> None:
+        """
+        Set the gripper position for a specific robot.
 
-        right_gripper_desired_joint = FOLLOWER_GRIPPER_JOINT_UNNORMALIZE_FN(
-            right_gripper_desired_pos_normalized
-        )
-        self.gripper_command.cmd = right_gripper_desired_joint
-        self.follower_bot_right.gripper.core.pub_single.publish(self.gripper_command)
+        :param robot_name: The name of the robot (e.g., 'follower_left' or 'follower_right').
+        :param gripper_desired_pos_normalized: The desired gripper position, normalized as a float value.
+        :return: None
+        """
+        # Unnormalize the gripper position
+        desired_gripper_joint = FOLLOWER_GRIPPER_JOINT_UNNORMALIZE_FN(
+            gripper_desired_pos_normalized)
+
+        # Update the gripper command with the unnormalized position
+        self.gripper_command.cmd = desired_gripper_joint
+
+        # Publish the command to the corresponding robot's gripper
+        self.robots[robot_name].gripper.core.pub_single.publish(
+            self.gripper_command)
 
     def _reset_joints(self):
         reset_position = START_ARM_POSE[:6]
```

Comment:
We are still using hardcoded values for constants is there a better way?

Reply:
I think it'd make sense to have this be a parameter, but we can get that on a second pass